content/components/data-table.mdx

```diff
@@ -600,6 +600,20 @@ The table must have a title, and can optionally have a subtitle to describe more
   src="https://user-images.githubusercontent.com/2313998/205361249-763772e1-d856-479e-a632-d185c27095db.png"
 />
 </Box>
+
+### Pagination buttons
+
+<Box as="p" flex={1}>
+  The pagination control for a DataTable uses `button` elements, and not `a` elements. This is because activating the 
```

Comment:
@joshblack - just want to confirm that DataTable pagination can _only_ ever be buttons, and there's no way for consumers to render links instead.

Reply:
Yup! Only buttons for now.